api/proto/apps/devflow/flow/flow.proto

```diff
@@ -2,59 +2,65 @@ syntax = "proto3";
 
 package erda.apps.devflow.flow;
 
-option go_package = "github.com/erda-project/erda-proto-go/apps/devflow/flow/pb";
-
-import "google/api/annotations.proto";
 import "common/openapi.proto";
+import "google/api/annotations.proto";
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
+option go_package = "github.com/erda-project/erda-proto-go/apps/devflow/flow/pb";
+
 service FlowService {
     option(erda.common.openapi_service) = {
-        service : "dop",
-        auth : {
-            check_login : true,
+        service: "dop",
+        auth: {
+            check_login: true,
             check_token: true,
         }
     };
-    rpc CreateFlowNode (CreateFlowNodeRequest) returns(CreateFlowNodeResponse) {
+    rpc CreateFlowNode(CreateFlowNodeRequest) returns (CreateFlowNodeResponse) {
         option(google.api.http) = {
-            post : "/api/devflow",
+            post: "/api/devflow",
         };
         option(erda.common.openapi) = {
-            path : "/api/devflow",
+            path: "/api/devflow",
         };
     }
-    rpc operationDeploy (OperationDeployRequest) returns (OperationDeployResponse) {
+    rpc OperationMerge(OperationMergeRequest) returns (OperationMergeResponse) {
         option(google.api.http) = {
-            put : "/api/devflow/{mergeID}/actions/operation-deploy",
+            put: "/api/devflow/{mergeID}/actions/operation-merge",
         };
         option(erda.common.openapi) = {
-            path : "/api/devflow/{mergeID}/actions/operation-deploy",
+            path: "/api/devflow/{mergeID}/actions/operation-merge",
         };
     }
-    rpc DeleteFlowNode (DeleteFlowNodeRequest) returns (DeleteFlowNodeResponse) {
+    rpc DeleteFlowNode(DeleteFlowNodeRequest) returns (DeleteFlowNodeResponse) {
         option(google.api.http) = {
-            delete : "/api/devflow/{mergeID}",
+            delete: "/api/devflow/{mergeID}",
         };
         option(erda.common.openapi) = {
-            path : "/api/devflow/{mergeID}",
+            path: "/api/devflow/{mergeID}",
         };
     }
-    rpc Reconstruction (ReconstructionRequest) returns (ReconstructionResponse) {
+    rpc Reconstruction(ReconstructionRequest) returns (ReconstructionResponse) {
         option(google.api.http) = {
-            post : "/api/devflow/actions/reconstruction",
+            post: "/api/devflow/actions/reconstruction",
         };
         option(erda.common.openapi) = {
-            path : "/api/devflow/actions/reconstruction",
+            path: "/api/devflow/actions/reconstruction",
         };
     }
-    rpc GetDevFlowInfo (GetDevFlowInfoRequest) returns (GetDevFlowInfoResponse) {
+    rpc GetDevFlowInfo(GetDevFlowInfoRequest) returns (GetDevFlowInfoResponse) {
         option(google.api.http) = {
-            get : "/api/devflow/actions/info",
+            get: "/api/devflow/actions/info",
```

Comment:
I think `/api/devflow/{flowID}` is more restful

Reply:
there are no flowID